extension/Descriptors/AssignmentCategoryDescriptor.xml

```diff
@@ -28,24 +28,24 @@
 		<CodeValue>COURSE_WORK_TYPE_UNSPECIFIED</CodeValue>
 		<ShortDescription>Unspecified</ShortDescription>
 		<Description>No work type specified. This is never returned.</Description>
-		<Namespace>uri://ed-fi.org/edfilms/AssignmentCategoryDescriptor/Google Classroom</Namespace>
+		<Namespace>uri://ed-fi.org/edfilms/AssignmentCategoryDescriptor/Google</Namespace>
```

Comment:
This was changed in an older PR and somehow the old version keeps coming back - this is now the third time I've tried to change it!